base/pkg/entry.jl

```diff
@@ -376,7 +376,11 @@ function update(branch::AbstractString)
                         LibGit2.fetch(repo)
                         LibGit2.merge!(repo, fastforward=true)
                     catch err
-                        show(err)
+                        if isa(err, LibGit2.GitError)
+                            print("Package $pkg cannot be updated. Error: $(err.msg) Class: $(err.class)")
+                        else
+                            show(err)
```

Comment:
All errors are reported on screen. Is there any problem with that? Why should the update process be stopped because one of the packages cannot be updated?